plugins/superio/README.md

```diff
@@ -0,0 +1,18 @@
+SuperIO
+=======
+
+This plugin enumerates the ITE8587 SuperIO embedded controller chip found in
+many rebadged "Clevo" laptops.
+
+See https://en.wikipedia.org/wiki/Super_I/O for more details about SuperIO
+and what the EC actually does.
+
+Eventually we will support flashing the EC using this plugin, but not until we
+have a way to recover a failed flash.
```

Comment:
The "normal" way to do this is to have a large SPI chip partitioned as 3 sections - bootloader, "A", and "B".  You don't normally touch bootloader, but swap between A and B and if you fail to boot one of them load from the other to recover.

Not knowing the intentions with this plugin; is this plugin going to translate into actions on "Clevo"'s side that they can adopt something like that?  Otherwise I'd worry about robustness of recovery.

Reply:
Really depends on the EC (and BIOS) architecture for the flash process.